specs/rollup-node.md

```diff
@@ -252,7 +254,7 @@ L1 Ethereum [reaches finality approximately every 12 minutes][l1-finality]. L2 b
 are "safer" than most recent L2 blocks because they will never disappear from the chain's history because of a re-org.
 However, they can still be challenged by a fault proof until the end of the fault proof window.
 
-[l1-finality]: https://www.paradigm.xyz/2021/07/ethereum-reorgs-after-the-merge/
+[l1-finality]: https://www.paradigm.xyz/2021/07/ethereum-reorgs-after-the-merge
```

Comment:
I made a minor edit just so I could comment here. 
1. I don't think this is a great resource to reference. It's probably not being kept up to date.
2. The [specs](https://github.com/ethereum/consensus-specs/blob/v1.0.0/specs/phase0/beacon-chain.md#time-parameters) give 6.4 minutes per epoch, so 12.8 minutes to finality. We should be precise.
3. This does not apply until after the merge. I think what we should do is try to estimate an L1 block depth that would have similar economic penalties to undoing a post-merge finalized consensus block.

Would appreciate thoughts on this tho.

Reply:
I think realistically Bedrock is going to go live after the merge, unless something bad and unforeseen happens. So I'm fine with speccing with the merge in mind, rather than having to do extra work which we then need to rip out.

I think this is better than no resources, but I think something more accurate would be much better indeed. Maybe referencing the L2 specs directly if there's a section we can point to?

Also the fact that finalization is 12.8 minutes but ONLY if 2/3 of nodes are online/operating is an important caveat.

Reply:
Updated to reference both this post, and the [eth2 specs](https://github.com/ethereum/consensus-specs/blob/v1.0.0/specs/phase0/beacon-chain.md#time-parameters).